crates/walrus-service/src/node/events/event_processor.rs

```diff
@@ -86,17 +86,17 @@ use crate::{
 };
 
 /// The name of the checkpoint store.
-const CHECKPOINT_STORE: &str = "checkpoint_store";
+pub const CHECKPOINT_STORE: &str = "checkpoint_store";
```

Comment:
nit: event store is exposed using `pub(crate) fn event_store_cf_name() -> &'static str {}` below. Maybe do the similar things for other CFs for consistency?

Reply:
That function seems a bit unnecessary tbh. I am not totally certain why adding another layer of encapsulation is better over just making it a pub const. If there isn't a strong reason, we can probably remove that other function.

Reply:
yeah. removing it is ok too. Let's just keep them consistent. (and maybe also change here to pub(crate)).

Reply:
I see, thanks for adding context @liquid-helium. Do you like [this](https://github.com/MystenLabs/walrus/pull/1940) approach of putting constants together in a mod names constants and then importing them as `use event_processor::constants::{self as event_processor_constants},` which makes it clear where these are being imported from. I